sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,284 @@
+                                                                                        .withPublicIPAddressConfiguration(
+                                                                                            new VirtualMachineScaleSetPublicIPAddressConfiguration()
+                                                                                                .withName(publicIpName)
+                                                                                                .withSku(
+                                                                                                    new PublicIPAddressSku()
+                                                                                                        .withName(PublicIPAddressSkuName.STANDARD)
+                                                                                                        .withTier(PublicIPAddressSkuTier.REGIONAL)
+                                                                                                )
+                                                                                                .withProperties(
+                                                                                                    new VirtualMachineScaleSetPublicIPAddressConfigurationProperties()
+                                                                                                        .withDeleteOption(DeleteOptions.DELETE)
+                                                                                                        .withPublicIPAddressVersion(IPVersion.IPV4)
+                                                                                                        .withIdleTimeoutInMinutes(9)
+                                                                                                        .withPublicIPPrefix(
+                                                                                                            new SubResource()
+                                                                                                                .withId(publicIpPrefix.id())
+                                                                                                        )
+                                                                                                )
+                                                                                        )
+                                                                                )
+                                                                        )
+                                                                    )
+                                                            )
+                                                    )
+                                                )
+                                                .withNetworkApiVersion(NetworkApiVersion.TWO_ZERO_TWO_ZERO_ONE_ONE_ZERO_ONE)
```

Comment:
2020-11-01 seems a bit old for NRP. Is this parameter required? Can we leave it out or specify latest version using `NetworkApiVersion.fromString`?

Reply:
Fixed in the new version.